src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/sw-settings-index.html.twig

```diff
@@ -19,6 +19,20 @@
                 </h1>
                 {% endblock %}
 
+                {# @deprecated tag:v6.8.0 - Block will be removed without replacement #}
+                {% block sw_settings_content_card_view_rename_banner %}
+                <mt-banner
+                    v-if="feature.isActive('v6.8.0.0') || !hideSettingRenameBanner"
+                    class="sw-settings__content-rename-banner"
+                    variant="info"
+                    closable
+                    @close="onCloseSettingRenameBanner"
+                >
+                    <p v-html="$t('sw-settings.index.textSettingRenameBanner')">
+                    </p>
+                </mt-banner>
+                {% endblock %}
```

Comment:
This is an incorrect implementation, let's imagine if the `v6.8.0.0` is active, the code will become as below.

```twig
{% block sw_settings_content_card_view_rename_banner %}
{% endblock %}
```